data/orders.yml

```diff
@@ -117,12 +117,12 @@
   state: planning
 - requesting_agency: Department of Agriculture
   office: Forest Service
-  rfq_id: null
+  rfq_id: 1160564
   title: Forest Service e-Permitting
-  solicitation_date: (2016-11)
-  period_of_performance: TBD
-  repository: null
-  description: null
+  solicitation_date: 2016-12-21
+  period_of_performance: Likely Jan-Feb 2017
+  repository: https://github.com/18F/bpa-fs-epermit-api
+  description: An application protocol interface (API) for the U.S. Forest Service ePermits special use permits.
   date_posted: 2016-10-27
-  date_updated: 2016-10-27
-  state: planning
+  date_updated: 2016-12-27
+  state: solicitation open
```

Comment:
The value the [template looks for](https://github.com/18F/ads-bpa/blob/18f-pages/pages/index.md#posted-for-bid) here is `posted`.